docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -156,13 +156,20 @@
                         others' checkpoints, and perhaps interfere with each other in other ways.
                     </td>
                 </tr>
-                <tr>
                     <td class="property" id="job-coordinator-system">job.coordinator.system</td>
                     <td class="default"></td>
                     <td class="description">
                         <strong>Required:</strong> The <span class="system">system-name</span> to use for creating and maintaining the <a href="../container/coordinator-stream.html">Coordinator Stream</a>.
                     </td>
                 </tr>
+                <tr>
+                    <td class="property" id="job-default-system">job.default.system</td>
```

Comment:
> systems.kafka-exact.samza.factory=org.apache.pkg.KafkaExactOnceSystemFactory
> systems.kafka-at-least.samza.factory=org.apache.pkg.KafkaAtLeastOnceSystemFactory
> systems.hdfs.samza.factory=org.apache.pkg.HdfsSystemFactory

Well, imo, exactly-once is a "property" of the producer. So, I would expect a configuration like this:
systems.kafka-exactly.samza.factory = org.apache.pkg.KafkaSystemFactory
systems.kafka-exactly.samza.producer.exactly-once.enabled = true

Factory includes a producer and consumer. I don't think we will introduce a separate factory for exactly-once producer. What does that mean for the consumer then? 
Also, I think we are all projecting what the config "might" look like. But my point was to distinguish system-scoped properties Vs stream-scoped properties. 

Another question : Based on your example, if I specify a producer property for a stream A and not stream B (which belongs to the same system), will we create 2 producer instances or 1 ?